CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # 6.0.0 (in progress)
 
+##### Bug fixes / Improvements
+* [#1768](https://github.com/oshi/oshi/pull/1768): Fix Incorrect use of Reference Equality - [@mythili-rajaraman](https://github.com/mythili-rajaraman).
```

Comment:
I think if I were Daniel, he would talk about the capitalization and past tense.

Reply:
```suggestion
* [#1768](https://github.com/oshi/oshi/pull/1768): Fixed incorrect use of reference equality - [@mythili-rajaraman](https://github.com/mythili-rajaraman).
```

Reply:
Totally. Done